client/zecale/cli/zecale_submit.py

```diff
@@ -2,23 +2,20 @@
 #
 # SPDX-License-Identifier: LGPL-3.0+
 
-from .utils import load_extended_proof
-from ..core.nested_transaction import NestedTransaction
-from click import option, command, pass_context, Context
+from .utils import load_nested_transaction
```

Comment:
What is the convention you follow for imports syntax? I can see that absolute paths are used as in `client/zecale/cli/utils.py` for instance, but here you use relative imports. Why not stick to absolute imports everywhere (the project is not very deep - at least for now..)

Reply:
I find it easier to deal with for groups of related files (e.g. groups of commands etc).  But no problem, it's trivial and actually got switched to absolute during the rebase.

Reply:
Thanks